components/searchable_user_list/searchable_user_list.jsx

```diff
@@ -216,7 +216,8 @@ export default class SearchableUserList extends React.Component {
                         id='searchUsersInput'
                         ref='filter'
                         className='form-control filter-textbox'
-                        placeholder={Utils.localizeMessage('filtered_user_list.search', 'Search users')}
+                        placeholder={{id: 'filtered_user_list.search', defaultMessage: 'Search users'}}
```

Comment:
Because we don't have any way to infer this from the `mmjstool i18nextract` tool, we should use here something like `{id: t('filtered_channels_list.search'), defaultMessage: 'Search channels'}`.